app-tracking-protection/vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/store/VpnDatabase.kt

```diff
@@ -204,6 +204,12 @@ abstract class VpnDatabase : RoomDatabase() {
             }
         }
 
+        private val MIGRATION_31_TO_32: Migration = object : Migration(31, 32) {
+            override fun migrate(database: SupportSQLiteDatabase) {
+                database.execSQL("ALTER TABLE `vpn_app_tracker_exclusion_list` ADD COLUMN `reason` TEXT NOT NULL DEFAULT 'UNKNOWN'")
```

Comment:
Given that we see crash increase every time we do a migration, and the current theory is that some users have a pretty full disk and we never vacuumed previous db tables improvements. Maybe we should do a `dataase.execSQL("VACUUM")` as first statement?

@marcosholgado what do you think?